### PR TITLE
Change de min lenght password, remove the symbols validate and remove obsolete import

### LINKED ISSUE #16: Fix password validation
The password validation must include

8 minimun lenght
20 maximun lenght
Must have uppercase letters
Must have lowercase letters
Must have digits

### PR DESCRIPTION
**Pull request by jlazo**
It include

- The min lenght for password is 8
- Ignore symbols in password
- Remove import in AuthService

### CODE REVIEW

File: src/modules/auth/auth.service.ts [hunk 1]
```diff
@@ -70,8 +69,8 @@ export class AuthService {
       .lowercase() // Must have lowercase letters
       .has()
       .digits() // Must have at least 2 digits
-      .has()
-      .symbols() // Must have symbols
+      // .has()
```
Comment (jcmexdev) [inline src/modules/auth/auth.service.ts]: I prefer remove that lines instead comment them what do you think? , it doesn't affect the functionality but I prefer don't send comment code to production.

**Review by jcmexdev — CHANGES_REQUESTED** [review]
I wrote some comments to you let me know what do you think, i'll be waiting your response to review again

File: src/modules/auth/auth.service.ts (hunk 1 above)
Comment (AlanDavd) [reply src/modules/auth/auth.service.ts]: I think comments as documentation or reference are great. We should focus on removing un used imports and unnecessary commented code

Comment (jlazo) [reply src/modules/auth/auth.service.ts]: Oks. Already

**Review by AlanDavd — APPROVED** [review]
LGTM

**Review by jcmexdev — APPROVED** [review]
It seems good 🚀

**Pull request merged**